scripts/Muon/GUI/Common/fitting_tab_widget/fitting_tab_model.py

```diff
@@ -7,28 +7,28 @@
 from Muon.GUI.Common.utilities.algorithm_utils import run_Fit, run_simultaneous_Fit
 import mantid
 from Muon.GUI.Common.ADSHandler.muon_workspace_wrapper import MuonWorkspaceWrapper
-from Muon.GUI.Common.ADSHandler.workspace_naming import get_fit_workspace_base_directory
+from Muon.GUI.Common.ADSHandler.workspace_naming import get_fit_workspace_directory
 from mantid.simpleapi import RenameWorkspace
-from mantid.api import FunctionFactory
 
 
 class FittingTabModel(object):
     def __init__(self):
         pass
 
     def do_single_fit(self, parameter_dict):
-        output_workspace, fitting_parameters_table, function_string, output_status, output_chi_squared =\
+        group_name = parameter_dict.pop('GroupName')
```

Comment:
why do we have this line?

Reply:
This is to remove the name of the ADS group from the dictionary before it is passed to the fit algorithm as this is not a property the algorithm will recognise